go.mod

```diff
@@ -70,7 +70,7 @@ require (
 	code.cloudfoundry.org/rep v0.0.0-20200325195957-1404b978e31e // indirect
 	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
 	code.cloudfoundry.org/tlsconfig v0.0.0-20200131000646-bbe0f8da39b3 // indirect
-	github.com/DataDog/agent-payload v0.0.0-20200416172112-13d2aa7ab86f // 4.31.0
+	github.com/DataDog/agent-payload v0.0.0-20200427190743-c19937ab1cc6 // mehedi.bakht/add-fields-for-dns-latency
```

Comment:
I know we haven't yet merged this to agent-payload. 

Do we need to change the way we tag agent-payload to reference this by tag?

Reply:
I just tried it
- Modified `go.mod` to use tags
`github.com/DataDog/agent-payload 4.33.0`
- then ran `inv -e deps`
- the line with tags got overwritten by the corresponding timestamp + commit hash
`github.com/DataDog/agent-payload v0.0.0-20200428185529-63c9964e037d`